### PR TITLE
Issue 95 multiple environments

### PR DESCRIPTION
**Pull request by elvismdev**
Handles #95

### CODE REVIEW

File: templates/page-settings.php [hunk 1]
```diff
@@ -5,99 +5,112 @@
...
-      <div class="wpcfm-error">Your PHP version is not compatible with Yaml export format. Upgrade to at least PHP 5.6.4.</div>
-    <?php endif; ?>
+    <?php if ( defined( 'WPCFM_CURRENT_ENV' ) && !empty( WPCFM_CURRENT_ENV ) ): ?>
+    <div class="wpcfm-info">
+        <?php _e( 'Current environment', 'wpcfm' ); ?>: <code><?php echo ucfirst( WPCFM_CURRENT_ENV ); ?></code>
+    </div>
+
+    <!-- Environment Switcher -->
+    <div class="wpcfm-env-switch">
+        <?php _e( 'Compare with', 'wpcfm' ); ?>: <select name="wpcfm_env_switch" id="wpcfm_env_switch">
```
Comment (ataylorme) [inline templates/page-settings.php]: I think `Compare with` is a bit confusing. While this is changing the current environment to you compare with `diff` it also changes what environment the `push` outputs as well

**Comment by ataylorme** [comment]
You can see a few commits I made in [this fork](https://github.com/ataylorme/wp-cfm/commits/master)

File: wp-cfm.php [hunk 2]
```diff
@@ -37,27 +37,92 @@ class WPCFM_Core
...
         define( 'WPCFM_DIR', dirname( __FILE__ ) );
-        define( 'WPCFM_CONFIG_DIR', apply_filters( 'wpcfm_config_dir', WP_CONTENT_DIR . '/config' ) );
-        define( 'WPCFM_CONFIG_URL', apply_filters( 'wpcfm_config_url', WP_CONTENT_URL . '/config' ) );
+
+        $config_dir = WP_CONTENT_DIR . '/config';
+        $config_url = WP_CONTENT_URL . '/config';
+
+        // Check if we are on Pantheon hosting environment.
+        if ( defined( 'PANTHEON_ENVIRONMENT' ) ) {
+            $this->pantheon_env = PANTHEON_ENVIRONMENT;
```
[the remark this answers is not in the document]
Comment (ataylorme) [reply wp-cfm.php]: We should also make `$this->pantheon_env` filterable so users can override this default logic with their own.

[the remark this answers is not in the document]
Comment (elvismdev) [reply wp-cfm.php]: Hi @ataylorme !

Thanks for the feedback. Will apply your recommended changes. 

The variable `$this->pantheon_env` can be filtered already with this: https://github.com/forumone/wp-cfm/blob/issue-95-multiple-environments/wp-cfm.php#L125

Is that what you are looking for?

[the remark this answers is not in the document]
Comment (ataylorme) [reply wp-cfm.php]: @elvismdev 👍 that works! Knowing how to use it is should be covered in the documentation

**Pull request merged**